### ISSUE TITLE
Add model eval loop to model selection loop...

### ISSUE THREAD

**Issue by jdurbin** [jdurbin/wekaMine#16]
for 5 cross validations:
   on training set do 5x cross validation search through models... pick best model
   evaluate on test set. 

Hmmm... but if different model picked each time??
